tests/users/test_views.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+from django.core import mail
+from django.urls import reverse
+from django.test import Client
+
+from vitrina.users.models import User
+
+
+@pytest.fixture
+def user():
+    user = User.objects.create_user(email="test@test.com", password="test123")
+    return user
+
+
+@pytest.mark.django_db
+def test_login_with_wrong_credentials(client: Client, user):
+    resp = client.post(reverse('login'), data={'email': "test@test.com", 'password': "wrongpassword"})
+    assert list(resp.context['form'].errors.values()) == [["Neteisingi prisijungimo duomenys"]]
+    assert resp.wsgi_request.user.is_authenticated is False
+    assert resp.wsgi_request.user.pk is None
+
+
+@pytest.mark.django_db
+def test_login_with_correct_credentials(client: Client, user):
+    resp = client.post(reverse('login'), data={'email': "test@test.com", 'password': "test123"})
+    assert resp.status_code == 302
+    assert resp.url == reverse('home')
+    assert resp.wsgi_request.user.is_authenticated is True
+    assert resp.wsgi_request.user.pk == user.pk
+
+
+@pytest.fixture
+def error_messages():
+    name_error = "Vardas ir pavardė negali būti trumpesni nei 3 simboliai, negali turėti skaičių"
+    email_error = "Naudotojas su tokiu el. pašto adresu jau egzistuoja"
+    password_error = "Slaptažodis turi būti ne trupesnis nei 8 simboliai, jį turi sudaryti raidės " \
+                     "ir skaičiai bei specialus simbolis"
+    terms_error = "Turite sutikti su naudojimo sąlygomis"
+    return {
+        'name_error': name_error,
+        'email_error': email_error,
+        'password_error': password_error,
+        'terms_error': terms_error
+    }
+
+
+@pytest.mark.django_db
+def test_register_with_short_name(client: Client, user, error_messages):
```

Comment:
```suggestion
def test_register_with_short_name(client: Client, error_messages):
```

`user` fixture is not used in this test.

Reply:
Could you also fix this for other tests?